README.md

```diff
@@ -54,7 +52,6 @@ experimenting with machine learning algorithms.
     ├── tests                   <- Folder for saving tests for the machine learning pipeline to make sure that
     │                              the source code works as expected.
     ├── .flake8                 <- Linter file necessary to format code to the OOT standards.
-    ├── .gitignore              <- List of the files not to add to this repository when committing to it.
```

Comment:
I don't mind having these here.

Reply:
This is total overkill. This is a git repo hosted on GitHub and we should explain what the `.gitignore` file is for? The docs are already bloated with too much information and I find them really hard to read. I can't imagine what that is like for a newcomer I would like to push for a radical simplification and IMHO it starts from removing stuff like this.